rules/integrations/o365/defense_evasion_microsoft_365_exchange_malware_filter_policy_deletion.toml

```diff
@@ -36,7 +37,7 @@ timestamp_override = "event.ingested"
 type = "query"
 
 query = '''
-event.dataset:o365.audit and event.provider:Exchange and event.category:web and event.action:"Remove-MalwareFilterPolicy" and event.outcome:success
+event.dataset:o365.audit and event.provider:Exchange and event.category:web and (event.action:"Remove-MalwareFilterPolicy" or event.action:"Disable-MalwareFilterRule") and event.outcome:success
```

Comment:
```suggestion
event.dataset:o365.audit and event.provider:Exchange and event.category:web and event.action:("Remove-MalwareFilterPolicy" or "Disable-MalwareFilterRule") and event.outcome:success
```

Reply:
I reverted this one, it is already covered in another rule